envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/EnvoySnapshotFactory.kt

```diff
@@ -217,8 +217,9 @@ class EnvoySnapshotFactory(
         globalSnapshot: GlobalSnapshot,
         egressRouteSpecifications: Collection<RouteSpecification>
     ): List<ClusterLoadAssignment> {
+        val resources = globalSnapshot.endpoints
```

Comment:
Why just endpoints are called resources?

Reply:
Old refactoring, I can call it endpoints.